lib/services.credentials.js

```diff
@@ -42,8 +45,30 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
       },
       */
       get: function(req, res, callback) {
-        var identityId = req.user.identity.id;
-        var idHash = database.hash(identityId);
+        // request is not authenticated
+        if(!req.user.identity && req.query.id) {
+          // FIXME: return a credential marked as public
+          // if no public crendentials, respond with 404
+          var publicQuery = {
+            'credential.id': req.query.id,
+            'credential.sysPublic': '*'
```

Comment:
We're short on time, so I'm going to merge this in. However, only after I add a TODO. This query won't capture credentials that aren't fully public, but we still need to display those credentials (just in part).